src/coreComponents/linearAlgebra/interfaces/hypre/HypreUtils.hpp

```diff
@@ -507,15 +508,16 @@ enum class MGRRestrictionType : HYPRE_Int
  */
 enum class MGRCoarseGridMethod : HYPRE_Int
 {
-  galerkin = 0,          //!< Galerkin coarse grid computation using RAP
-  nonGalerkin = 1,       //!< Non-Galerkin coarse grid computation with dropping strategy: inv(A_FF) approximated by its (block) diagonal
-                         //!< inverse
-  cprLikeDiag = 2,       //!< Non-Galerkin coarse grid computation with dropping strategy: CPR-like approximation with inv(A_FF)
-                         //!< approximated by its diagonal inverse
-  cprLikeBlockDiag = 3,  //!< Non-Galerkin coarse grid computation with dropping strategy: CPR-like approximation with inv(A_FF)
-                         //!< approximated by its block diagonal inverse
-  approximateInverse = 4 //!< Non-Galerkin coarse grid computation with dropping strategy: inv(A_FF) approximated by sparse approximate
-                         //!< inverse
+  galerkin = 0,           //!< Galerkin coarse grid computation using RAP
+  nonGalerkin = 1,        //!< Non-Galerkin coarse grid computation with dropping strategy: inv(A_FF) approximated by its (block) diagonal
+                          //!< inverse
+  cprLikeDiag = 2,        //!< Non-Galerkin coarse grid computation with dropping strategy: CPR-like approximation with inv(A_FF)
+                          //!< approximated by its diagonal inverse
+  cprLikeBlockDiag = 3,   //!< Non-Galerkin coarse grid computation with dropping strategy: CPR-like approximation with inv(A_FF)
+                          //!< approximated by its block diagonal inverse
+  approximateInverse = 4, //!< Non-Galerkin coarse grid computation with dropping strategy: inv(A_FF) approximated by sparse approximate
+                          //!< inverse
+  nonGalerkinRAI = 5      //!< Non-Galerkin coarse grid computation with arbitrary classical restriction and injective prolongation
```

Comment:
@castelletto1 I think it's better to call this `galerkinRAI` because this isn't a non-Galerkin approximation, it's just a cheaper way of doing the galerkin product. Do you agree?